petl/test/io/test_xlsx.py

```diff
@@ -11,229 +11,218 @@
 from petl.io.xlsx import fromxlsx, toxlsx, appendxlsx
 from petl.test.helpers import ieq, eq_
 
+openpyxl = pytest.importorskip("openpyxl")
```

Comment:
Code scanning:
Constant name "openpyxl" doesn't conform to UPPER_CASE naming style

Reply:
That's the suggested way by pytest, I guess pylint does not go well with pytest here

Reply:
- `pytest` does agree with flake, pylint and his friends?
- Anyway we'll have to deal with 5000+ of these :)